caasp-velero-image/config.sh

```diff
@@ -16,6 +16,6 @@ echo "Configure image: [$kiwi_iname]..."
 # velero/velero
 # same file structure
 #--------------------------------------
-cp /usr/bin/velero /velero
+ln -sf /usr/bin/velero /velero
```

Comment:
Do we need to do the same for velero-restic-restore-helper?

Reply:
Good question, I cannot find code in velero with hardcode path `/velero-restic-restore-helper`.
However, for sync with upstream, it's good to do the same for `velero-restic-restore-helper`.

Reply:
Not sure if I understand about this one. If the entrypoint remains in `<<entrypoint execute="velero"/>` and velero package already installs in executable path /usr/bin, why need the extra symbolic link?